rosidl_typesupport_introspection_c/include/rosidl_typesupport_introspection_c/message_introspection.h

```diff
@@ -38,6 +38,8 @@ typedef struct rosidl_typesupport_introspection_c__MessageMember_s
   size_t (* size_function)(const void *);
   const void * (*get_const_function)(const void *, size_t index);
   void * (*get_function)(void *, size_t index);
+  void (* fetch_function)(const void *, size_t index, void *);
```

Comment:
Can we add a short description to these new functions here? This way we save a bit of time iterating on https://github.com/ros2/rosidl/pull/646.

Reply:
Done in https://github.com/ros2/rosidl/pull/652/commits/42b67c8cd72655d101e5a1f7a9d2d4d776f0cd26.